### PR TITLE
Change admin vote text field to a radio button form; Restrict input type for votes to Numbers

### LINKED ISSUE #1649: Voter Dapp: simplify admin proposal voting
The voter dapp should show a "yes" or "no" choice rather than making the user type 1 or 0.

### LINKED ISSUE #1665: Limit voter dapp price entry field to only take numbers

### PR DESCRIPTION
**Pull request by nicholaspai**
- Non-admin vote entry restricted to numbers. Resolves #1665 
- Two helper scripts: one to propose an Admin action, one to execute an approved admin action
- Voter dApp checks if a given price request is an admin proposal. If it is, it replaces the TextField with a Form that contains two Radio buttons: "YES" and "NO" corresponding to "1" and "0". Note that the price displayed for committed admin votes is now translated for the user as YES or NO. Resolves #1649 

How I tested:
- Voted with 2Key
- Voted without 2Key
- After revealing both admin and non-admin votes, I was able to retrieve rewards and subsequently execute the admin proposal

Screenshots:
- How price requests look pre-commit. Note that for admin votes, non-"1" and "1" are translated to "NO" and "YES" respectively. For example, if I vote "123" via the CLI, then the vote will appear as "NO" on the dApp.
Screen Shot 2020-07-01 at 11 47 53
- Entering a value to be committed. Note the restriction to numbers-only input for non-admin vote, and the radio buttons for admin votes.
Screen Shot 2020-07-01 at 11 48 08
- How price requests look post-commit.
Screen Shot 2020-07-01 at 11 48 22
- How price requests look post-reveal.
Screen Shot 2020-07-01 at 11 48 57
- Successfully retrieving rewards for resolved votes via the CLI
Screen Shot 2020-07-01 at 11 49 39
- Resolved requests appearing on the dApp
Screen Shot 2020-07-01 at 11 49 45
- Executing the approved admin proposal via a new script
Screen Shot 2020-07-01 at 11 50 03
- How the admin requests look on a really really small screen 
Screen Shot 2020-07-01 at 11 53 25

### CODE REVIEW

File: core/scripts/local/ExecuteAdmin.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    const governor = await Governor.deployed();
+
+    // Get the latest admin proposal
+    const proposalId = (await governor.numProposals()).subn(1).toString();
+    const proposal = await governor.getProposal(proposalId);
+
+    // for every transactions within the proposal
+    for (let i = 0; i < proposal.transactions.length; i++) {
+      console.log(`${i}: Submitting tx...`);
+      let tx = await governor.executeProposal(proposalId.toString(), i.toString());
```
Comment (mrice32) [inline core/scripts/local/ExecuteAdmin.js:20]: nit: `const`

Comment (nicholaspai) [reply core/scripts/local/ExecuteAdmin.js:20]: > Looks great! Thoughts on modifying the votes in the resolved requests table so they say YES for nonzero values and NO for zero values?

Ah yes, will do!

Comment (nicholaspai) [reply core/scripts/local/ExecuteAdmin.js:20]: > Looks great! Thoughts on modifying the votes in the resolved requests table so they say YES for nonzero values and NO for zero values?

Done here. Also in the process of testing this I was able to test out the new feature where you can't retrieve rewards if you voted incorrectly!
Screen Shot 2020-07-01 at 15 46 06

Comment (mrice32) [reply core/scripts/local/ExecuteAdmin.js:20]: very nice :)

File: common/AdminUtils.js [hunk 2]
```diff
@@ -37,8 +37,14 @@ function getAdminRequestId(identifierUtf8) {
   return parseInt(identifierUtf8.slice(adminPrefix.length), 10);
 }
 
+// Vote 1 for Yes, anything else for No.
+const translateAdminVote = voteValue => {
+  return voteValue === "1" ? "YES" : "NO";
```
Comment (mrice32) [inline common/AdminUtils.js:42]: nit: I think this is _slightly_ inaccurate. Technically, the Governor interprets any nonzero value as YES.

See https://github.com/UMAprotocol/protocol/blob/b04c2e2fa1f94709e9952a8728c3e7b7ef91244f/core/contracts/oracle/implementation/Governor.sol#L153.

However, if someone manually votes `3` (outside of the dapp) and everyone else votes `1`, we don't want to classify those both as YES in the dapp leading the voter to think they voted correctly and deserve rewards. All that is to say, maybe we should do something like this:
```
switch(voteValue) {
  case "1":
    return "YES";
  case "0":
    return "NO";
  default:
    return "INVALID ADMIN VOTE";
}
```

Thoughts?

Comment (nicholaspai) [reply common/AdminUtils.js:42]: So let me confirm my understanding:
- any non zero answer is a YES, but that means there many different ways to say YES
- only one YES value is the correct vote (I.e. “1” is YES and is the correct vote, while “3” is also YES but is not the correct vote)

If this is correct then I agree with your 3 cases

Comment (mrice32) [reply common/AdminUtils.js:42]: That’s correct.

Comment (nicholaspai) [reply common/AdminUtils.js]: Implemented

File: common/AdminUtils.js [hunk 3]
```diff
@@ -37,8 +37,21 @@ function getAdminRequestId(identifierUtf8) {
   return parseInt(identifierUtf8.slice(adminPrefix.length), 10);
 }
 
+// Vote 1 for Yes, anything else for No.
```
Comment (mrice32) [inline common/AdminUtils.js:40]: nit: outdated comment

**Pull request merged**